src/Components/Search/Previews/Grids/ArtistSearch/MarketingCollections.tsx

```diff
@@ -14,9 +14,14 @@ interface MarketingCollectionsPreviewProps {
   searchState?: SearchBarState
 }
 
-const CollectionBox = styled(Box)<{ imageUrl: string }>`
+const CollectionBox = styled(Box)<{ imageUrl: string; itemsPerRow: 1 | 2 }>`
```

Comment:
The CollectionBox needs to know how many items are in each row, so it can apply alternating margins to account for the space in the middle.